examples/java/src/main/java/org/apache/beam/examples/WindowedWordCount.java

```diff
@@ -72,21 +72,20 @@
  * }
  * </pre>
  *
- * <p>To execute this pipeline using the Dataflow service, specify pipeline configuration:
+ * <p>To change the runner, specify:
```

Comment:
I think this should say something like "By default, the examples will run with the DirectRunner. To change the runner, specify:"